content/packages/learn-github-packages/configuring-a-packages-access-control-and-visibility.md

```diff
@@ -152,7 +152,7 @@ You can choose the visibility of containers that organization members can publis
 6. Under "Container creation", choose whether you want to enable the creation of public, private, or internal container images.
     - To enable organization members to create public container images, click **Public**.
     - To enable organization members to create private container images that are only visible to other organization members, click **Private**. You can further customize the visibility of private container images.
-    - **For {% data variables.product.prodname_ghe_cloud %} only:** To enable organization members to create internal container images that are only visible to other organization members, click **Internal**.
+    - To enable organization members to create internal container images that are visible to all organization members, click **Internal**. Incase where the owning organization belongs to an enterprise, it will be visible to all enterprise members.
```

Comment:
```suggestion
    - To enable organization members to create internal container images that are visible to all organization members, click **Internal**. In cases where the owning organization belongs to an enterprise, it will be visible to all enterprise members.
```
I think this is a typo, but if it isn't, feel free to ignore this

Reply:
I think it would be clearer to say:
```suggestion
    - To enable organization members to create internal container images that are visible to all organization members, click **Internal**. If the organization belongs to an enterprise, the container images will be visible to all enterprise members.
```

Reply:
@hubwriter I would definitely get behind your suggestion, it does make it a lot clearer.